zcash_client_sqlite/CHANGELOG.md

```diff
@@ -16,6 +16,16 @@ and this library adheres to Rust's notion of
 - Migrated from `schemer` to our fork `schemerz`.
 - Migrated to `rusqlite 0.32`.
 
+### Fixed
+- `zcash_client_sqlite::WalletDb`'s implementation of
+  `zcash_client_backend::data_api::WalletRead::get_wallet_summary` has been
+  fixed to take account of `min_confirmations` for transparent balances.
```

Comment:
```suggestion
  fixed to take account of `min_confirmations` for transparent balances.
  (Previously, it would treat transparent balances as though `min_confirmations`
  were `1` even if it was set to a higher value.)
```